src/test/java/integration/ITest.java

```diff
@@ -0,0 +1,20 @@
+package integration;
+
+import static com.codeborne.selenide.Selenide.closeWebDriver;
+
+import com.codeborne.selenide.Configuration;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+
+public class ITest {
+    @BeforeAll
+    static void beforeAll() {
+        Configuration.baseUrl = "https://borisosipov.github.io/test-page";
```

Comment:
@BorisOsipov Maybe move this testing page to https://selenide-selenoid.github.io?

Reply:
@asolntsev Sure. Please enable github pages for branch `gh-pages` on this repo settings

Reply:
@BorisOsipov I have enabled it: https://selenide.org/selenide-selenoid/

P.S. Probably it's better to have some landing page (or at least some welcome text) on the page https://selenide.org/selenide-selenoid/, and put testing pages  to some subfolder etc.

Reply:
I updated the tests to run against https://selenide.org/test-page/